array/DNAm/analysis/EWAS/lmWithinCT.r

```diff
@@ -0,0 +1,117 @@
+##---------------------------------------------------------------------#
+##
+## Title: EWAS using linear regression model within each cell type
+##
+## Purpose of script: perform DNA methylation association analysis of 
+## schizophrenia vs controls for each cell type separately and 
+## including relevant cell composition covariates
+##
+##---------------------------------------------------------------------#
+
+
+#----------------------------------------------------------------------#
+# DEFINE ANALYSIS FUNCTION

```

Comment:
thought for future - have analysis functions as a seperate script which is then sourced here so that the user can specify their own variables etc. And specify it in the config file